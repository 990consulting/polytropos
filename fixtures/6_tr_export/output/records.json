{
  "record_0": {
    "1": {
      "boolean_like": {
        "target_binary": true,
        "target_unary": null
      },
      "target_string": "ABC"
    },
    "2": {
      "boolean_like": {
        "target_binary": false,
        "target_unary": true
      },
      "target_string": "DEF"
    },
    "invariant": {
      "outer_list": [
        {
          "inner_list": [
            {"name": "A_1_1_1"},
            {"name": "A_1_1_2"}
          ]
        },
        {
          "inner_list": [
            {"name": "B_1_2_1"},
            {"name": "B_1_2_2"}
          ]
        }
      ],
      "some_other_field": "X"
    }
  },
  "record_1": {
    "1": {
      "boolean_like": {
        "target_binary": true,
        "target_unary": true
      },
      "target_string": "GHI"
    },
    "invariant": {
      "outer_list": [
        {
          "inner_list": [
            {"name": "C_1_1_1"},
            {"name": "C_1_1_2"}
          ]
        },
        {
          "inner_list": [
            {"name": "D_1_2_1"},
            {"name": "D_1_2_2"}
          ]
        }
      ],
      "some_other_field": "Y"
    }
  }
}