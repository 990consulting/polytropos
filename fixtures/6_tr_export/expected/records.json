{
  "record_0": {
    "1": {
      "boolean_like": {
        "target_binary": true,
        "target_unary": null
      },
      "target_string": "ABC"
    },
    "2": {
      "boolean_like": {
        "target_binary": false,
        "target_unary": true
      },
      "target_string": "DEF"
    },
    "invariant": {
      "outer_list": [
        {
          "inner_list": [
            {
              "name": "B_1_1_1"
            },
            {
              "name": "B_1_1_2"
            }
          ]
        },
        {
          "inner_list": [
            {
              "name": "A_2_1_1"
            },
            {
              "name": "A_2_1_2"
            }
          ]
        }
      ],
      "target_random_text": "X"
    }
  },
  "record_1": {
    "1": {
      "boolean_like": {
        "target_binary": true,
        "target_unary": true
      },
      "target_string": "GHI"
    },
    "invariant": {
      "outer_list": [
        {
          "inner_list": [
            {
              "name": "D_1_1_1"
            },
            {
              "name": "D_1_1_2"
            }
          ]
        },
        {
          "inner_list": [
            {
              "name": "C_2_1_1"
            },
            {
              "name": "C_2_1_2"
            }
          ]
        }
      ],
      "target_random_text": "Y"
    }
  }
}